completion/available/fabric.completion.bash

```diff
@@ -79,13 +81,13 @@ function __fab_completion() {
 
 	# Variables to hold the current word and possible matches
 	local cur="${COMP_WORDS[COMP_CWORD]}"
-	local opts=()
+	local opts
 
 	# Generate possible matches and store them in variable "opts"
 	case "${cur}" in
 		-*)
 			if [[ -z "${__FAB_COMPLETION_LONG_OPT}" ]]; then
-				export __FAB_COMPLETION_LONG_OPT=$(
+				__FAB_COMPLETION_LONG_OPT=$(
 					fab --help | grep -E -o "\-\-[A-Za-z_\-]+\=?" | sort -u
 				)
```

Comment:
https://github.com/Bash-it/bash-it/pull/2294#discussion_r2009069723

Sorry for my confusion.

```suggestion
				)
				export __FAB_COMPLETION_LONG_OPT
```